deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/ContainersTable.tsx

```diff
@@ -71,14 +72,18 @@ const loader = async ({ request }: LoaderFunctionArgs): Promise<LoaderData> => {
     [])[0] ?? null) as MalwareScanGroupedStatus | null;
 
   const order = getOrderFromSearchParams(searchParams);
-
+  const hosts = searchParams.get('hosts')?.split(',') ?? [];
+  const filterIn: {
+    [key: string]: string[];
+  } = hosts.length ? { host_name: hosts } : {};
   const searchSearchNodeReq: SearchSearchNodeReq = {
     node_filter: {
       filters: {
         compare_filter: null,
         contains_filter: {
           filter_in: {
             active: [true],
+            ...filterIn,
```

Comment:
why create filterIn variable, if it is not being reused? we can just put value directly here.

Reply:
removed local var and added inline conditional logic